src/components/EventCard/index.jsx

```diff
@@ -3,17 +3,29 @@ import PropTypes from 'prop-types';
 import React, { useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
 
+import { isURL, getAbsoluteURL } from '../../utils';
+
 import './styles.less';
 
 const EventCard = props => {
   const history = useHistory();
+
+  let location = <p className="location">{props.location}</p>;
+  if (isURL(props.location)) {
+    location = (
+      <a className="link" href={getAbsoluteURL(props.location)}>
+        {location}
+      </a>
+    );
```

Comment:
why not condense this to a ternary operator inside the jsx?

Reply:
Yep, much cleaner!